File: 2 Factory/src/MYSQL.java
public class MYSQL implements IDB {
    @Override
    public void selectData() {
        System.out.println("MYSQL veritabanından veri okundu.");
    }
    
    @Override
    public void insertData() {
        System.out.println("MYSQL veritabanına veri yazıldı.");
    }
}

File: 2 Factory/chsarp/MYSQL.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Factory_csharp
{
    public class MYSQL : IDB
    {
        public void SelectData()
        {
            Console.WriteLine("MYSQL veritabanından veri okundu.");
        }

        public void InsertData()
        {
            Console.WriteLine("MYSQL veritabanına veri yazıldı.");
        }
    }
}
